src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/components/ClinicalBoxPlot.js

```diff
@@ -141,7 +141,7 @@ const ClinicalBoxPlot = ({
         style={{
           height: CHART_HEIGHT + 10,
           maxHeight: CHART_HEIGHT + 10,
-          width: QQ_PLOT_RATIO,
+          width: 'calc(100% - 150px)',
```

Comment:
calculates the available space dynamically, rather than relying on a fixed percentage (which doesn't account for the boxplot's fixed width)